src/main/java/com/javachallengers/simpson/model/dto/SimpsonCharacterRequestDTO.java

```diff
@@ -0,0 +1,24 @@
+package com.javachallengers.simpson.model.dto;
+
+import java.io.Serializable;
+
+public final class SimpsonCharacterRequestDTO implements Serializable {
+	private static final long serialVersionUID = 8840536784759714847L;
+
+	private final String name;
```

Comment:
Can we add more fields here so it's possible to add the SimpsonCharacterRequestDTO with all front-end information?

Reply:
Yes, we can add more fields.

Reply:
I added birthDate, City and Country fields.